src/pynxtools_raman_multiformat/reader.py

```diff
@@ -60,24 +83,101 @@ def handle_eln_file(self, file_path: str) -> Dict[str, Any]:
             convert_dict=CONVERT_DICT,
             parent_key="/ENTRY[entry]",
         )
-        #self.txt_line_skips = self.eln_data.get('/ENTRY[entry]/skip')
 
         return {}
 
-    def handle_txt_file(self, filepath) -> Dict[str, Any]:
-        self.read_txt_file(filepath)
-        return {}
-
     def get_attr(self, key: str, path: str) -> Any:
         """
-        Get the metadata that was stored in the main(=data) file.
+        Get the metadata that was stored in the main file.
         """
+        return self.get_metadata(self.raman_data, path, self.callbacks.entry_name)
 
-        if self.txt_data is None:
+    def get_eln_data(self, key: str, path: str) -> Any:

```

Comment:
This should be removed, is a copy of the same function as above

Reply:
upsi. right.